FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Services/AuthConfiguration.swift

```diff
@@ -6,17 +6,17 @@ public struct AuthConfiguration {
   let interactiveDismissEnabled: Bool
   let shouldAutoUpgradeAnonymousUsers: Bool
   let customStringsBundle: Bundle?
-  let tosUrl: URL
-  let privacyPolicyUrl: URL
+  let tosUrl: URL?
+  let privacyPolicyUrl: URL?
```

Comment:
nit: with the behavior in the view rendering an empty view if one but not both are set, we should document that both these properties need to be set in order to render a privacy and terms footer.

Reply:
Good catch, I've updated README here: [`37903bd` (#1254)](https://github.com/firebase/FirebaseUI-iOS/pull/1254/commits/37903bded3f61a0923836911754905b079207a6a)